format-specs/geoparquet.md

```diff
@@ -80,10 +80,10 @@ which itself implements the model of
 Apart from the difference of encodings, the semantics are intended to match
 WKT2:2019, and a CRS in one encoding can generally be represented in the other.
 
-If CRS is not provided, all coordinates in the geometry must use longitude, latitude
-based on the WGS84 datum to store their data.
+If the `crs` field is not provided, the default CRS is [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84),
+which means that all coordinates in the geometries must use longitude, latitude based on the WGS84 datum.
```

Comment:
```suggestion
which means CRS-aware implementations can assume a default value of [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84)
```

I think switching the order as suggested in these two lines can get across that if you are implementing and don't know about CRS's then you can just use long/lat. I do think it'd be good to not make it seem like to implement geoparquet you need a full projection engine - you could just read and write only long/lat. 

If we were to do a full OGC-style spec we'd likely have the core conformance just be long/lat, and another conformance class add in more projections (crs field and ability to understand it)

Reply:
We tried to avoid the "can assume" wording as this is weird, so another proposal:

```suggestion
which means the default value is [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84) for CRS-aware implementations
```